spec/rails_helper.rb

```diff
@@ -22,7 +21,7 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  # config.use_transactional_fixtures = true
```

Comment:
Any reason not to start with this on?  Is it because we don't actually have a DB yet?

Reply:
yep

Reply:
https://github.com/sul-dlss/pre-assembly/issues/145